src/app/api/composio/connections/route.ts

```
@@ -7,17 +7,17 @@ import { getComposioClient } from "@/lib/composio"; // Updated import
-  created_at: string;
-  updated_at: string;
-  scopes?: string[];
-  metadata?: Record<string, unknown>;
-  // Any other fields returned by the v3 SDK's list method
+  id: string;
+  status: "ACTIVE" | "EXPIRED" | "INITIATED" | "INITIALIZING" | "FAILED"; // As per error
+  toolkit: { slug: string }; // As per error (actual type has toolkit.slug)
+  data: Record<string, unknown>; // As per error (actual type has data)
+  createdAt: string; // As per error (camelCase)
+  updatedAt: string; // As per error (camelCase)
+  authConfig: { id: string; [key: string]: any }; // As per error (actual type has authConfig.id for auth_config_id)
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)